proxy/http/HttpTransact.cc

```diff
@@ -558,7 +558,8 @@ HttpTransact::BadRequest(State *s)
 void
 HttpTransact::HandleBlindTunnel(State *s)
 {
-  bool inbound_transparent_p = s->state_machine->ua_session->get_netvc()->get_is_transparent();
+  NetVConnection *vc         = s->state_machine->ua_session->get_netvc();
+  bool inbound_transparent_p = vc->get_is_transparent();
```

Comment:
This change isn't going to protect from a NULL vc dereference.  I just double checked through and it is the change that was made on the master branch, so the backport is consistent.